mino/minogrpc/server.go

```diff
@@ -11,13 +11,12 @@ import (
 	"context"
 	"crypto"
 	"crypto/ecdsa"
-	"crypto/rand"
 	"crypto/tls"
 	"crypto/x509"
+	"google.golang.org/grpc/credentials/insecure"
+	"strings"
```

Comment:
we're using goimports formatting, which isn't respected here.

Reply:
I'm never sure what Goland does, sorry. I'll try to convince it to use `goimports`. Do you have a common formatter you're using?